### PR TITLE
[release-0.15] Annotate tests with polarion IDs

### PR DESCRIPTION
**Pull request by phoracek**
**What this PR does / why we need it**:

In order to be able to track test results with Polarion, we need to add these annotations.

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/main_test.go [hunk 1]
```diff
@@ -72,9 +73,15 @@ func TestE2E(tapi *testing.T) {
...
-	RunSpecsWithDefaultAndCustomReporters(t, "E2E Test Suite", []Reporter{junitReporter, knmstateReporter})
+
+	reporters := make([]Reporter, 0)
+	reporters = append(reporters, knmstatereporter.New("test_logs/e2e", framework.Global.Namespace, nodes))
+	reporters = append(reporters, ginkgoreporters.NewJUnitReporter("junit.functest.xml"))
+	if qereporters.JunitOutput != "" {
+		reporters = append(reporters, qereporters.NewJunitReporter())
+	}
+
+	RunSpecsWithDefaultAndCustomReporters(t, "E2E Test Suite", reporters)
```
Comment (qinqon) [inline test/e2e/main_test.go]: Backport the fix

**Comment by qinqon** [comment]
/hold

We have to re-backport the fix

File: test/e2e/main_test.go (hunk 1 above)
Comment (qinqon) [reply test/e2e/main_test.go]: @phoracek added a commit https://github.com/nmstate/kubernetes-nmstate/pull/453/commits/fa54c38ad342a549ee58c690778715ce698c6782 to address this.

**Comment by qinqon** [comment]
/hold cancel

Everything is backported now.

File: test/e2e/rollback_test.go [hunk 2]
```diff
@@ -61,7 +61,7 @@ var _ = Describe("rollback", func() {
 	})
 	// This spec is done only at first node since policy has to be different
 	// per node (ip addresses has to be different at cluster).
-	Context("when connectivity to default gw is lost after state configuration", func() {
+	Context("[test_id:3793]when connectivity to default gw is lost after state configuration", func() {
```
Comment (phoracek) [inline test/e2e/rollback_test.go]: This should be at `It`. This was a bug on the master PR

Comment (qinqon) [reply test/e2e/rollback_test.go]: ok, done.

**Comment by qinqon** [comment]
Looks like junit artifacts are not being generated

```
PASS
ok  	github.com/nmstate/kubernetes-nmstate/test/e2e	1034.697s
time="2020-03-30T10:20:31Z" level=info msg="Local operator test successfully completed."
+ teardown
+ make cluster-down
kubevirtci/cluster-up//down.sh
++ find . -name '*junit*.xml'
+ cp /logs/artifacts
cp: missing destination file operand after '/logs/artifacts'
Try 'cp --help' for more information.
+ EXIT_VALUE=1
+ set +o xtrace
Cleaning up after docker in docker.
======
```

**Pull request merged**